.gitignore

```diff
@@ -0,0 +1,48 @@
+# Compiled Object files
```

Comment:
Is this file included by accident?

Reply:
No. Copied from rocBLAS intentionally. Should I remove it?

Reply:
It's ok to keep it